Add frequency detector

landauermax opened the pull request:
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [X] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [X] Issues exist for this PR
- [X] I added related issues using the "Fixes #123'-notations
- [X] This Pull-Requests merges into the "development"-branch

Fixes #399

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [X] I have updated the documentation accordingly
- [X] I have added tests to cover my changes
- [X] All new and existing tests passed

# Describe changes:
- Add an analsis component that detects deviations in occurrence frequencies

4cti0nfi9ure reviewed (changes requested):
Please explain/define the confidence_factor in the code and the documentation. It is not clear what it exactly does and why it is a value greater than 1 and not in the interval [0,1].

landauermax commented:
> 
> 
> Please explain/define the confidence_factor in the code and the documentation. It is not clear what it exactly does and why it is a value greater than 1 and not in the interval [0,1].

You are correct, it makes more sense to use the parameter in normalized form. I changed it to lie in the interval [0, 1] and updated the documentation in code and in the wiki.

4cti0nfi9ure reviewed (approved):
looks good.
